crates/ide_completion/src/lib.rs

```diff
@@ -147,6 +147,7 @@ pub fn completions(
     let ctx = CompletionContext::new(db, position, config)?;
 
     let mut acc = Completions::default();
+    completions::infered_type(&mut acc, &ctx);
```

Comment:
```suggestion
    completions::inferred_type(&mut acc, &ctx);
```
let's put this `completions::format_string::format_string(&mut acc, &ctx);` so this remains sorted